Documentation/Concepts/event_store.md

```diff
@@ -2,90 +2,189 @@
-
-## Querying the Event Store
-
-Event Stores are not general purpose data storage engines.  As such they require and provide only rudimentary querying capabilities.
-
-### Rehydrating an Event Source
-
-The most common requirement to query an Event Store is to re-populate an Event Source or Aggregate Root.  This simply involves retrieving all the commits for this Event Source (identified by the Event Source Id) and re-applying them.  For performance optimisation reasons, this can be extended to include a commit version such that you only retrieve commits from a specific version number.  This is relevant when a snapshot is available that aggregates all previous events up to the specific version and you only have to apply that and subsequent commits.  When the Event Source is stateless (i.e. it does not need to keep any state to maintain its invariants) we bypass the loading of commits completely and just set the version directly.
-
-### Catching up an Event Processor
-
-An event processor operates at the Event rather than the Commit level, which only exists on the write-side of the event-store.  When an event processor is instantiated it checks if there have been any events of the type that it handles since the last one it handled (in the case of a new event processor this would typically be since the beginning of time).  As an optimisation the Event Store can be asked to provide all instance of a particular event type since a particular **Committed Event Version**.
-
-### Catching up Event Horizons
-
-Similar to Event Processors, Event Horizons have to catch up since the last version they processed.  An Event Horizon operates on Commits rather than a single event type (though it splits the commit and passes it to the appropriate Barrier).  Therefore it is possible to ask the Event Store for all Commits (across Event Sources) that have occurred since a particular version.
-
-## Snapshots
-
-[to be added]
-
-
-## Event Migration
-
-[to be added]
+An Event Store is a database optimized for storing [Events]({{< ref "events" >}}) in an [Event Sourced]({{< ref "event_sourcing" >}}) system. The [Runtime]({{< ref "overview" >}}) manages the connections and structure of the stored data. All [Streams]({{< ref "streams" >}}), [Event Handlers & Filters]({{< ref "event_handlers_and_filters" >}}), [Aggregates]({{< ref "aggregates" >}}) and [Event Horizon Subscriptions]({{< ref "event_horizon#subscription" >}}) are being kept track inside the event store.
+
+The event store is an append-only database. Any events saved to it **cannot be changed** or **deleted**. It acts as the record of all events that have happened in the system from the beginning of time.
```

Comment:
This may or may not be kind of confusing. Since the Event Store is the culmination of multiple collections, not just "streams". So it is not completely correct to say that the event store is "append-only"

Reply:
```suggestion
Events saved to the event store **cannot be changed** or **deleted**. It acts as the record of all events that have happened in the system from the beginning of time. 
```
I removed the "append-only" part.